zebra-state/src/service/check.rs

```diff
@@ -316,7 +321,7 @@ where
 
         // If we are past our NU5 activation height, but there are no V5 transactions in recent blocks,
         // the Zebra instance that verified those blocks had no NU5 activation height.
```

Comment:
Shouldn't the `Zebra instance` be `other peer`?
```suggestion
        // the other peer that verified those blocks had no NU5 activation height.
```

Reply:
We're checking the local cached state here, without any network connections:
```suggestion
        // the last Zebra instance that updated this cached state had no NU5 activation height.
```